featurebyte/schema/worker/task/feature_list_create.py

```diff
@@ -35,22 +35,32 @@ def feature_params_discriminator(value: Any) -> Literal["feature_ids", "feature_
     -------
     Literal["feature_ids", "feature_params"]
     """
-    if isinstance(value, list):
-        if value:
-            return feature_params_discriminator(value[0])
+    if not value:
         return "feature_ids"
+
     if isinstance(value, (dict, FeatureParameters)):
         return "feature_params"
+
+    while isinstance(value, list):
```

Comment:
How does update to starlette affect the existing logic here?

Reply:
Let me revert this, I was debugging whether it was the discriminator or the class typing that was causing pydantic issues.
It was the class schema in the end.